docs/guides/advanced/v8_migration_guide.md

```diff
@@ -260,7 +279,10 @@ public void ConfigureServices(IServiceCollection services)
     {
         busCfg.AddEntityFrameworkOutbox<MyDbContext>(outboxCfg =>
         {
-            outboxCfg.UseSqlServer(); // Use your database flavor
+            // outboxCfg.UseSqlServer(); // Use your database flavor
+            // NOTE: Mass Transit's lock statement providers are currently incompatible with EF8, so for now use this instead:
+            outboxCfg.LockStatementProvider =
```

Comment:
Just add a note that this is 'termporary'

Reply:
I was hoping that `for now` would be clear enough? We can revert this once MT fixes it and we remove custom lock providers from our code (naturally forcing replacement).

Reply:
Or I can just… not touch this and hopefully let it resolve itself in a few weeks.